libs/natparse/src/main/java/org/amshove/natparse/lexing/Lexer.java

```diff
@@ -891,20 +910,58 @@ private boolean mightBeDataFormat(String possibleDataFormat)
 		return floatingPointCount < 2;
 	}
 
+	private boolean consumeNaturalHeader()
+	{
+		if (isAtLineStart() && (isSingleAsteriskComment() || isInlineComment()))
```

Comment:
You can simplify this `if`/`else` with
```
		if (!isAtLineStart() || (!isSingleAsteriskComment() && !isInlineComment()))
		{
			return false;
		}
```

Reply:
Would it be better to put the negation outside instead?
if (!(isAtLineStart() || (isSingleAsteriskComment() && isInlineComment()))